grug/app/src/app.rs

```diff
@@ -1,5 +1,7 @@
+use crate::NaiveQuerier;
 #[cfg(feature = "abci")]
 use grug_types::{JsonDeExt, JsonSerExt};
+
```

Comment:
Can you put the `use crate::NaiveQuerier` into the big `use` statement below?

If you have the editor generate the `use` statement here it messes up our formatting, and rustfmt isn't smart enough to fix this.